README.adoc

```diff
@@ -384,51 +367,63 @@ include::{common-includes}/mvncompile.adoc[]
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Create the `ConfigurationTest` class.#
-`src/test/java/it/io/openliberty/guides/config/ConfigurationTest.java`
+#Create the `ConfigurationIT` class.#
+`src/test/java/it/io/openliberty/guides/config/ConfigurationIT.java`
 ----
-ConfigurationTest.java
+ConfigurationIT.java
 [source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/config/ConfigurationTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/config/ConfigurationIT.java[]
 ----
 
-The [hotspot=testInitialServiceStatus file=0]`testInitialServiceStatus()` test case reads the value of the `io_openliberty_guides_inventory_inMaintenance` configuration property in the file `META-INF/microprofile-config.properties` and checks the HTTP response of the inventory service.
+microprofile-config.properties
+[source, properties, linenums, role='code_column']
+----
+include::finish/src/main/resources/META-INF/microprofile-config.properties[]
+----
+
+The [hotspot=testInitialServiceStatus file=0]`testInitialServiceStatus()` test case reads the value of the [hotspot=inventory-in-maintenance file=1]`io_openliberty_guides_inventory_inMaintenance` configuration property in the file [hotspot file=1]`META-INF/microprofile-config.properties` and checks the HTTP response of the inventory service.
```

Comment:
Change
"The [hotspot=testInitialServiceStatus file=0]`testInitialServiceStatus()` test case reads the value of the [hotspot=inventory-in-maintenance file=1]`io_openliberty_guides_inventory_inMaintenance` configuration property in the file [hotspot file=1]`META-INF/microprofile-config.properties` and checks the HTTP response of the inventory service."
To
"The [hotspot=testInitialServiceStatus file=0]`testInitialServiceStatus()` test case reads the value of the [hotspot=inventory-in-maintenance file=1]`io_openliberty_guides_inventory_inMaintenance` configuration property in the [hotspot file=1]`META-INF/microprofile-config.properties` file and checks the HTTP response of the `inventory` service."

Reply:
Updated